### DISCUSSION TITLE
Way to reset lfs storage

### DISCUSSION THREAD

**Discussion by vLesio** [community/community#42794] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi, today I got an email regarding usage of lfs storage, saying it has been blocked because our organization used more than 150% of available data. Surprised, I looked at billings page and noticed that one of our projects pushed 2GB of lfs files. It has been done by mistake (well, guy who did this didn't know he shouldn't because he missed my messages regarding this topic).

Is there any way that this usage can be restored to previous state? I already removed an entire branch from repo.

I know this 'storage' is based on push actions, but I'm asking just in case something can be done to undo this mistake.

**Comment by airtower-luna** [comment, marked as answer]
According to the documentation, there's no easy way to remove unused LFS objects. The advice given there is to either delete and recreate the repository, or contact support: https://docs.github.com/en/repositories/working-with-files/managing-large-files/removing-files-from-git-large-file-storage#git-lfs-objects-in-your-repository :confused:

**Reply by vLesio** [reply]
Thank you, I somehow missed this page in the documentation.
